src/form-question-types.rst

```diff
@@ -1655,33 +1655,86 @@ In the parameters column, write ``max-pixels=`` followed by the desired maximum
 Audio widget
 ----------------
 
-Records audio from an external app.
+type
+  :tc:`audio`
+appearance
+  :tc:`none`
 
-.. note::
+Records audio using the device's microphone.
 
-  Some Android devices provide a default application for audio recording.
-  Others do not,
-  and the user will need to install an audio recording app.
+.. image:: /img/form-widgets/audio-start.*
+  :alt: The Audio form widget as displayed in the ODK Collect App on an Android phone. The question text is "What does it sound like?" There are two buttons: Record Sound and Choose Sound.
 
-  There are many apps available for this, including:
+.. rubric:: XLSForm
 
-  - `Axet Audio Recorder (open source) <https://play.google.com/store/apps/details?id=com.github.axet.audiorecorder>`_
-  - `RecForge II <https://play.google.com/store/apps/details?id=dje073.android.modernrecforge>`_
+.. csv-table:: survey
+  :header: type, name, label
 
-  Any app that responds to
-  `android.provider.MediaStore.Audio.Media.RECORD_SOUND_ACTION`
-  will be compatible.
+  audio, sound_like, What does it sound like?
 
 
-.. image:: /img/form-widgets/audio-start.*
-  :alt: The Audio form widget as displayed in the ODK Collect App on an Android phone. The question text is "Please record your name." There are three buttons: Record Sound, Choose Sound and Play Sound. The "Play Sound" button is disabled.
+Customizing audio quality
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The quality of audio recordings can be customized using the ``quality`` parameter. The available quality values are:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Value
+     - Extension
+     - Encoding
+     - Bit rate
+     - Sample rate
+     - File size
+   * - normal
+     - .m4a
+     - AAC
+     - 64kbps
+     - 32kHz
+     - ~30MB/hour
+   * - voice-only
+     - .amr
+     - AMR
+     - 12.2kbps
+     - 8kHz
+     - ~5MB/hour
+
+.. tip::
+
+  We'd recommend only using ``voice-only`` for one-on-one interviews in a quiet place as otherwise there might be too much detail loss. If in doubt, it's a good idea to test the different qualities out to see which one fits your use case best.
+
+  If it's a possibility that an individual question could need different qualities depending on context you can use :ref:`relevance <relevants>` to switch between them.
```

Comment:
I think this is not going to be actionable for most people. I wonder whether for things like this we might consider adding support questions that we answer ourselves and then link to them? Alternately we could use a folder in the `getodk` Google Drive to have form definitions we link to.

Reply:
This has been discussed a few times right? Like a gallery of example forms. I think if they are docs they should probably live as part of the docs code. Should we just do an inline example here?

Reply:
> if they are docs they should probably live as part of the docs code

I see that. How about having it inline but collapsed by default? Like "show details" in https://docs.getodk.org/briefcase-install/, for example (though I wouldn't say that's a very compelling use of the feature). So you can keep exactly the text you have now as a new paragraph and then maybe "show worked example" or even just "show details" right below?

Reply:
To be explicit, my concern is that we kind of want users to be able to skim vertically down the page and get everything that there is to get about audio recording. If there's a very long example related to an uncommon case then they may get distracted and miss important things below.

I think longer-term we should consider splitting up this very long page so topics are a bit better contained. But the collapsing of long examples seems like a fine approach for now.

Reply:
> How about having it inline but collapsed by default?

I like that idea! Will go with that.